From f105681ee28cd779a7451f324f23d142bc020c87 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Wed, 14 Oct 2015 13:00:10 +0300
Subject: [PATCH 1/3] clearfog: switch and external phy reset deassert
Organization: O.S. Systems Software LTDA.

R87 which is a pull up on the clearfog board will be removed in order to force
reset of the switch and external 1512 phy that is connected to the switch.

The forced reset is with a pull down that is already on the board.

With that MPP41 which previously was set as an input doesn't deassert the reset
signal since R87 pullup is now removed; so MPP41 must be driven high to force
reset deassert.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 board/mv_ebu/a38x/armada_38x_family/boardEnv/mvBoardEnvSpec38x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/mv_ebu/a38x/armada_38x_family/boardEnv/mvBoardEnvSpec38x.c b/board/mv_ebu/a38x/armada_38x_family/boardEnv/mvBoardEnvSpec38x.c
index 87e4ad3..ba62ca5 100644
--- a/board/mv_ebu/a38x/armada_38x_family/boardEnv/mvBoardEnvSpec38x.c
+++ b/board/mv_ebu/a38x/armada_38x_family/boardEnv/mvBoardEnvSpec38x.c
@@ -297,7 +297,7 @@ void a38x_customer_board_1_gpp_callback(MV_BOARD_INFO *board) {
 	MV_REG_BIT_RESET (0x18140, 0x200);
 	MV_REG_BIT_RESET (0x18144, 0x200);
 	mvOsDelay(1);
-	MV_REG_BIT_SET (0x18144, 0x200);
+	MV_REG_BIT_SET (0x18140, 0x200);
 	mvOsDelay(10);
 	for (i = 0; i < board->numIoExp; i++) {
 		if (MV_OK != mvBoardTwsiSet(BOARD_TWSI_IO_EXPANDER, board->pIoExp[i].addr,
-- 
2.7.1

